introducing-java/src/main/java/com/onto/javacourse/intro/gcd/GCD.java

```diff
@@ -1,5 +1,33 @@
-public class Main {
-    public static void main(String[] args) {
+public class GCD {
 
+    /**
+     * Method to find GCD
+     */
+    public static int GCD(int a, int b) {
+        if (a != 0 || b != 0) {
+            while (a != b) {
+                if (a > b) {
+                    a -= b;
+                } else {
+                    b -= a;
+                }
+            }
+            return a;
+        } else {
+            throw new IllegalArgumentException("Invalid numbers");
```

Comment:
This is good, but I would swap the if the other way around. This way the code will be less nested in brackets.